app/views/user/profile.php

```diff
@@ -38,19 +50,9 @@ class="profile-img">
                 <small>@ <?= htmlspecialchars($profile->getUsername() ?? 'Unknown'); ?></small>
 
                 <p class="bio"><?= htmlspecialchars($profile->getBio() ?? 'This user has not set a bio yet.'); ?></p>
-                <?php if ($userId !== $profileUserId): ?> <!-- Only show follow/unfollow buttons if viewing another user's profile -->
-                    <?php if (isset($isFollowing) && $isFollowing): ?>
-                        <form action="/user/unfollow/<?= htmlspecialchars($profileUserId); ?>" method="POST">
-                            <button type="submit" class="btn btn-danger">Unfollow</button>
-                        </form>
-                    <?php else: ?>
-                        <form action="/user/follow/<?= htmlspecialchars($profileUserId); ?>" method="POST">
-                            <button type="submit" class="btn btn-primary">Follow</button>
-                        </form>
-                    <?php endif; ?>
-                <?php endif; ?>
                 <div class="follow-stats">
                     <small>Following: <?= htmlspecialchars($userProfile->getFollowingCount() ?? 0); ?></small>
+                    <i>|</i>
```

Comment:
what's this i tag doing?

Reply:
Separating the following and follewers by |

Following:12 | Followers:34

Reply:
it doesnt need to be in a icon tag

Reply:
I'm blind, I thought you wrote "what is this | doing". totally misread it
Fixed: I have changed it do divs instead. Yes it doesn't need that either, but for the sake of it not blending with other vertical lines in VS Code, I decided to keep it in div tags